region/norfair/east/Volcano Room.json

```diff
@@ -160,17 +175,25 @@
           {"and": [
             "canCarefulJump",
             "canMockball",
-            {"lavaFrames": 180}
+            {"lavaFrames": 160}
           ]},
           {"lavaFrames": 220}
         ]},
         {"or": [
           {"and": [
             "canSpaceJumpWaterBounce",
-            {"lavaFrames": 120}
+            {"lavaFrames": 60}
           ]},
           {"lavaFrames": 150}
         ]}
+      ],
+      "exitCondition": {
+        "leaveNormally": {}
+      },
+      "unlocksDoors": [
+        {"types": ["missiles"], "requires": [{"lavaFrames": 80}]},
+        {"types": ["super"], "requires": []},
```

Comment:
This added lava frames when spacejumping.  The fastest way I saw with a blue door was to do an aim down shot to open the door.